### PR TITLE
[CHI-382] Added label inline positioning

### PR DESCRIPTION
**Pull request by janibolkvadze**
Added inline positioning of label
Added different examples to the documentation

@jllr @mattnickles 

https://ctl.atlassian.net/browse/CHI-382

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/306/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/307/ for more details.

File: src/chi/components/input-wrapper.scss [hunk 1]
```diff
@@ -22,6 +22,21 @@ $border: 0.0625rem;
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: start;
+    margin-bottom: 0.5rem;
+    label {
+      margin-bottom: 0;
+      padding-right: 0.5rem;
+    }
+    input {
```
Comment (jllr) [inline src/chi/components/input-wrapper.scss]: It is no necessary, please remove

File: src/chi/components/input-wrapper.scss [hunk 2]
```diff
@@ -22,6 +22,21 @@ $border: 0.0625rem;
     }
   }
 
+  &__inline {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: start;
+    margin-bottom: 0.5rem;
```
Comment (jllr) [inline src/chi/components/input-wrapper.scss]: This must be sorted alphabetically

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/311/ for more details.

**Comment by jllr** [comment]
@janibolkvadze please solve conflicts

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/324/ for more details.

**Pull request closed without merging**